packages/@lwc/template-compiler/src/__tests__/fixtures/attributes/scoped-id/static-vs-non-static-optimized/static-optimized/expected.js

```diff
@@ -0,0 +1,80 @@
+import _implicitStylesheets from "./static-optimized.css";
+import _implicitScopedStylesheets from "./static-optimized.scoped.css?scoped=true";
+import { freezeTemplate, parseFragment, registerTemplate } from "lwc";
+const $fragment1 = parseFragment`<section class="these-are-dynamic${0}"${2}><div${"a1:aria-describedby"}${"a1:aria-activedescendant"}${"a1:aria-errormessage"}${"a1:aria-flowto"}${"a1:aria-labelledby"}${"a1:for"}${"a1:id"}${3}></div></section>`;
+const $fragment2 = parseFragment`<section class="these-are-static${0}"${2}><div${"a1:aria-describedby"}${"a1:aria-activedescendant"}${"a1:aria-errormessage"}${"a1:aria-flowto"}${"a1:aria-labelledby"}${"a1:for"}${"a1:id"}${3}></div></section>`;
+const $fragment3 = parseFragment`<section class="these-are-boolean-true${0}"${2}><div aria-describedby aria-activedescendant aria-errormessage aria-flowto aria-labelledby for id${3}></div></section>`;
+const $fragment4 = parseFragment`<section class="these-are-the-empty-string${0}"${2}><div${"a1:aria-describedby"}${"a1:aria-activedescendant"}${"a1:aria-errormessage"}${"a1:aria-flowto"}${"a1:aria-labelledby"}${"a1:for"}${"a1:id"}${3}></div></section>`;
```

Comment:
This is the test where I am trying to confirm identical behavior between static-optimized and non-static-optimized (i.e. no observable changes). Note that "boolean true" is treated differently from "empty string."